examples/scripts/postCreateCommand.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
As this and requirements.txt is purely part of the devcontainer setup, perhaps we can group the 3 files related to the devcontainer together so that it's clear that they belong to each other?

examples/devcontainer/devcontainer.json
examples/devcontainer/requirements.txt
examples/devcontainer/post_create_command.sh